deployment/v3/mosip/resident/install.sh

```diff
@@ -12,29 +12,30 @@ CHART_VERSION=12.0.2
 echo Create $NS namespace
 kubectl create ns $NS
 
-echo Istio label 
+echo Istio label
 kubectl label ns $NS istio-injection=enabled --overwrite
 helm repo update
 
 echo Copy configmaps
 ./copy_cm.sh
 
-echo Installing Resident
-helm -n $NS install resident mosip/resident --version $CHART_VERSION
-
 API_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-api-internal-host})
-resident-ui_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-resident-ui-host})
+resident_ui_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-resident-ui-host})
```

Comment:
Need the below changes
* Use caps and remove UI
    RESIDENT_HOST
    mosip-resident-host